pg2mysql.pl

```diff
@@ -216,28 +236,37 @@ sub handle_create {
     $line =~ s/ \S*\.citext/ text/;
 
     my $field_def = ( $line !~ m/^CREATE/ && $line !~ m/^\s*CONSTRAINT/ && $line !~ m/\s*PRIMARY KEY/ && $line !~ m/^\s*\);/ );
-    
+
+    # revert rename special columns
```

Comment:
Again, I'm confused here.

Reply:
Anyway, the previous rename is reverted here, after converting all operations for the column instructions. So nothing is really changed :)